### PR TITLE
Refactor content to be fluid

### PR DESCRIPTION
**Pull request by manolobevia**
#31 explains how I made fluid content

### CODE REVIEW

File: components/oku-circ.vue [hunk 1]
```diff
@@ -78,6 +78,18 @@ export default {
 $light-blue: #3C96D2;
 $red: #D0011B;
 
+// Width scale factor for Olin
+$olin-width: 0.053vw;
```
Comment (cappadona) [inline components/oku-circ.vue]: Maybe make these more generic like: `lw`, `lh`, `pw`, `ph`, for landscape and portrait width and height?

Comment (cappadona) [inline components/oku-circ.vue]: Could go with `l-width`, `l-height` if the 2-letter abbreviations are too extreme ;)

Comment (manolobevia) [inline components/oku-circ.vue]: - 2 letters works.
- _re:  I'm seeing some overlap and tightness when testing it here_. Let me check.
- Notice the vertical alignment of the count within the circles is not vertical aligned to the center.

**Comment by cappadona** [comment]
Thanks...here's a local screenshot of Uris. Uris phone chargers are a bit cramped...you seeing the same?

localhost-3000-oku-uris hdtv

**Comment by manolobevia** [comment]
macbook pro
tv
ipnone 6
I think now Olin display looks good. Please review.

**Pull request merged**